rebasehelper/logger.py

```diff
@@ -21,10 +21,47 @@
 #          Tomas Hozza <thozza@redhat.com>
 
 import logging
+import six
 
 import rebasehelper.utils
 
 
+SUCCESS = logging.INFO + 5
+HEADING = logging.INFO + 6
+IMPORTANT = logging.INFO + 7
+
+_nameToLevel = {
+    'SUCCESS': SUCCESS,
+    'HEADING': HEADING,
+    'IMPORTANT': IMPORTANT,
+}
+
+
+class CustomLogger(logging.Logger):
+
+    SUCCESS = SUCCESS
+    HEADING = HEADING
+    IMPORTANT = IMPORTANT
```

Comment:
What's this? ;-) Why don't you just move the constants and `_nameToLevel` dict here?